src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,72 @@
+package com.elixr.training;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+                if(args.length!=2)
+                {
+                    System.out.println("enter the valid path");
+                    return;
+                }
+                String inputfile = args[0];
```

Comment:
Its a filepath, not file. Please change the variable name as needed.

Reply:
changed method name to inputfilepath